pgjdbc/src/main/java/org/postgresql/core/v3/ConnectionFactoryImpl.java

```diff
@@ -237,7 +237,7 @@ public QueryExecutor openConnectionImpl(HostSpec[] hostSpecs, String user, Strin
         // we trap this an return a more meaningful message for the end user
         GlobalHostStatusTracker.reportHostStatus(hostSpec, HostStatus.ConnectFail);
         knownStates.put(hostSpec, HostStatus.ConnectFail);
-        log(Level.WARNING, "ConnectException occurred while connecting to {0}", cex, hostSpec);
+        log(Level.FINE, "ConnectException occurred while connecting to {0}", cex, hostSpec);
```

Comment:
-1
The tricky part with this one is the exception is never shown in case of multi-URL configuration.
What do you think if log level depends on the "re-throwability"?
For instance: WARNING for not re-thrown error, and FINE for re-thrown error.

Reply:
Well, in this particular case, the exception is re-throw, so it should be FINE.